docs/modules/ROOT/pages/concepts/registry.adoc

```diff
@@ -3,6 +3,8 @@
 
 A `Meter` is the interface for collecting a set of measurements (which we individually call metrics) about your application. Meters in Micrometer are created from and held in a `MeterRegistry`. Each supported monitoring system has an implementation of `MeterRegistry`. How a registry is created varies for each implementation.
 
+TIP: Recoding a measurement for a `Meter` is expected to be a simple math/tallying operation that will not consume any significant amount of computing time or throw exceptions. Transferring metrics to an external monitoring system is a scheduled operation on a separate thread and will not interfere with meter recordings.
```

Comment:
I would simplify the first part to something along the lines "recording a measurement for a `Meter` is expected to be a relatively cheap operation and it should not throw any exception."
The second sentence is about publishing which is a bit implementation dependent. Perhaps I would word it as something like "if the registry supports publishing metrics to a metrics backend, this should be done on a separate thread from recording metrics"

There's also a bit of a question of whether both these belong in this page (Registry) of the docs. Maybe we should move the Meters page to before the Registry page in the table of contents and have the first part about recording measurements there?

Reply:
> I would simplify the first part to something along the lines "recording a measurement for a Meter is expected to be a relatively cheap operation and it should not throw any exception."
The second sentence is about publishing which is a bit implementation dependent. Perhaps I would word it as something like "if the registry supports publishing metrics to a metrics backend, this should be done on a separate thread from recording metrics"

Sounds good, I will rephrase according to your suggestion.

> There's also a bit of a question of whether both these belong in this page (Registry) of the docs. Maybe we should move the Meters page to before the Registry page in the table of contents and have the first part about recording measurements there?

Interesting. That might be nice. Let me try and move things around a bit and then we can see how it looks.

Reply:
@shakuzen Is this what you had in mind?

Reply:
Yes, that looks good to me. I left a minor review comment.